ses-resources/cephadm

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+
+# Source this to run cephadm-specific validations
+# for the SUSE Enterprise Storage supportutils plugin
+
+
+#############################################################
+rpm_list="$SES_SES_RESOURCES_DIR"/rpm-list
+for thisrpm in $(cat "$rpm_list"); do
+    validate_rpm_if_installed "$thisrpm"
+done
+
```

Comment:
The entire block above here belongs in the main `ses` script, as rpm validation should happen for both the cephadm and rook cases.

Reply:
Are we sure that's the case? Kubernetes nodes won't need the same RPMs installed. 

I will change this, because I do see that it could be useful to validate RPMs on k8s nodes sometimes, but I think most of the time, the RPM validation won't tell very much about Rook clusters.

Reply:
You're right, the RPM validation won't tell us much about the rook side of thinks, but, the RPMs in this list are all the RPMs we ship on the SES media, so regardless of whether we're using k8s or cephadm, if any of them are installed on the host, we want to know about them.